tests/i18n/tests.py

```diff
@@ -1918,10 +1918,18 @@ class NonDjangoLanguageTests(SimpleTestCase):
         LANGUAGES=[
             ('en-us', 'English'),
             ('xxx', 'Somelanguage'),
+            ('xyz', 'XYZ'),
```

Comment:
Actually xxx has a directory under `command/locale` and so the error isn't raised. The error is raised only when the locale directory doesn't exist.